src/illumidesk/authenticators/handlers.py

```diff
@@ -98,6 +98,7 @@ def post(self):
         lti_utils = LTIUtils()
         validator = LTI13LaunchValidator()
         args = lti_utils.convert_request_to_dict(self.request.arguments)
+        self.log.debug('Initial login request args are %s' % args)
         if validator.validate_authentication_request(args):
```

Comment:
shouldn't we replace it with the new method `validate_login_request`?

Reply:
Yes. I’ll add a test for this to ensure it stays that way.